curriculum/challenges/english/17-college-algebra-with-python/build-a-multi-function-calculator/multi-function-calculator.md

```diff
@@ -0,0 +1,29 @@
+---
+id: 63d83ff239c73468b059cd3f
+title: "Multi-Function Calculator"
+challengeType: 10
+dashedName: multi-function-calculator
+---
+
+# --description--
+
+You will be <a href="https://colab.research.google.com/drive/1jT7atuRCOnkdPyDmlgKVJqxouDSx7Ioh?usp=sharing" target="_blank" rel="noopener noreferrer nofollow">working on this project with Google Colaboratory</a>.
+
+After going to that link, create a copy of the notebook either in your own account or locally. Once you complete the project and it passes the test (included at that link), submit your project link below. If you are submitting a Google Colaboratory link, make sure to turn on link sharing for "anyone with the link."
+
+# --instructions--
+
+For this challenge, you need to create a multi-function calculator using Python that take input and do the following:
+
+- solve proportions
+- solve for x in equations
+- factor square roots
+- convert decimals to fractions and percents
+- convert fractions to decimals and percents
+- convert percents to decimals and fractions
+
+If you are struggling, you can follow the <a href="https://www.youtube.com/embed/PdsvcZNPEEs" target="_blank" rel="noopener noreferrer nofollow">video walkthrough for this project.</a>
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```

Comment:
This should be removed, or should be `# --hints--`
```suggestion
```

Reply:
<img width="472" alt="image" src="https://user-images.githubusercontent.com/51722130/224116882-38cd934e-73cf-4fa4-bcdd-0513b8c849ee.png">